docs/user/orion.rst

```diff
@@ -0,0 +1,97 @@
+
+Installation
+^^^^^^^^^^^^
+
+Orion is installed like every other plugin via IntelliJ's marketplace. Alternatively, although not usually needed, it is possible to install builds directly from `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_. The installation process is described in the `readme <https://github.com/ls1intum/Orion#testing-of-pull-requests>`_.
+
+General Usage
+^^^^^^^^^^^^^
+
+After installation, Orion provides a custom tool window at *View -> Tool Windows -> Artemis*. This window contains an integrated browser displaying Artemis.
+
+At the top of the tool window is a back button |back-button|. Clicking it returns to the initial page, that is:
+
+- The exercise details page if an exercise is opened as a student.
+- The exercise edit in editor page if an exercise is opened as an instructor.
+- The assessment dashboard if an exercise is opened as a tutor, or, if a submission is downloaded, the assessment details page of that submission.
+- The Artemis home page if no Artemis project is opened.
+
+Settings
+^^^^^^^^
+
+Orion's settings are at *Settings -> Tools -> Orion*. The settings include:
+
+- The Artemis base url. Can be changed to switch to a specific Artemis instance. Defaults to https://artemis.ase.in.tum.de. **Important:** The url must not end with a /, otherwise it does not work!
+- The Artemis exercise paths. Orion suggests to store newly downloaded exercises at ``default-path/course/exercise-name``, with the default path dependant of the setting.
+- The default commit message
+- An option to change the user agent if needed. The user agent is sent to Artemis to identify Orion. I needs to contain "Orion/*orion-version*". Usually, no changes are required.
+- A button to restart the integrated browser. If Orion gets stuck, restarting the browser might solve the issue without restarting IntelliJ.
+
+Problems
+^^^^^^^^
+
+Please report any issues on `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_.
+
+Participation in Orion
+----------------------
+
+Students can participate in programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise using the integrated browser.
+ 2. After starting the exercise, click |open-in-intellij-button|.
+ 3. After cloning, the exercise can be solved in IntelliJ.
+ 4. To submit the changes, click |submit-button|. This automatically commits, pushes the local changes, and displays the result of the automatic tests in IntelliJ.
+
+Gif showcasing the usage for students:
+
+  .. image:: https://raw.githubusercontent.com/ls1intum/Orion/master/.github/media/orion_workflow.gif
+            :align: center
+            :width: 100%
+
+Exercise Creation in Orion
+--------------------------
+
+Instructors can set up programming exercises via Orion by performing the following steps:
+
+ 1. The exercise needs to be created as described at the exercise creation of programming exercises, step 1 and 2.
```

Comment:
can we add a link here?

Reply:
I tried that, but it failed, see https://github.com/ls1intum/Artemis/pull/4044#discussion_r718518803

Reply:
Would it be an option to add an absolute link that references the release documentation? => It will work on release?